src/tv2-common/cues/ekstern.ts

```diff
@@ -59,10 +59,7 @@ export function EvaluateEksternBase<
 	adlib?: boolean,
 	rank?: number
 ) {
-	const eksternProps = parsedCue.source
-		.replace(/\s+/i, ' ')
-		.trim()
-		.match(/^(?:LIVE|SKYPE|FEED) ?([^\s]+)(?: (.+))?$/i)
+	const eksternProps = parsedCue.source.match(/^(?:LIVE|SKYPE|FEED) ?([^\s]+)(?: (.+))?$/i)
```

Comment:
It would be helpful if you could extract the regex matcher and explain what it matches:
```
const explainWhatItIsMatching: RegExp = /^(?:LIVE|SKYPE|FEED) ?([^\s]+)(?: (.+))?$/i
const eksternProps = parsedCue.source.match(explainWhatItIsMatching)
```

Reply:
It would be even better if we didn't need to parse the same thing all over again 😃 
@thomasslee97 what do you say if `source` in `CueDefinitionEkstern` became an object containing the already trimmed and parsed: `type`, `input`, `params`, and perhaps the `rawSource` just in case.

Reply:
I think this is future work under the scope of moving to a better source parsing workflow